src/backend/TrafficCourts/Staff.Service/Controllers/DisputeController.cs

```diff
@@ -101,12 +101,12 @@ public async Task<IActionResult> GetDisputeAsync(Guid disputeId, CancellationTok
         }
         catch (ApiException e)
         {
-            _logger.LogError("Error retrieving Dispute from oracle-data-api:", e);
+            _logger.LogError(e, "Error retrieving Dispute from oracle-data-api");

```

Comment:
Yeah, these are all from having Java developers code in C# :)
In Java, the parameters are reversed.